src/compiler.rs

```diff
@@ -125,7 +125,8 @@ impl AmberCompiler {
         }
         result.push(block.translate(&mut meta));
         let res = result.join("\n");
-        format!("{}\n{}", include_str!("header.sh"), res)
+        let header = include_str!("header.sh");
+        format!("{}\n{}", &header.replace("|version|", option_env!("CARGO_PKG_VERSION").unwrap()), res)
```

Comment:
Given that we use [chrono](https://docs.rs/chrono/latest/chrono/) for dates I propose the following changes:

```suggestion
        let header = [
            include_str!("header.sh"),
            "version:", option_env!("CARGO_PKG_VERSION").unwrap(),
            "date:", Local::now().format("%Y-%m-%d %H:%M:%S").to_string() 
        ].join(" ");
        format!("{}\n{}", header, res)
```